src/components/SideBar/SideBar.stories.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import { SideBar } from './SideBar';
+
+export default {
+  title: 'Components/Side Bar',
+  component: SideBar,
+} as ComponentMeta<typeof SideBar>;
+
+const Template: ComponentStory<typeof SideBar> = (args) => (
+  <SideBar {...args} />
+);
+
+export const MainSideBar = Template.bind({});
+MainSideBar.args = {
+  lessonCardsList: [
+    {
```

Comment:
This list looks like a duplicate of the one in `src/components/Pages/MainPage/MainPage.stories.tsx`

Does it make more sense to make this more modular? If not, then no need to waste time doing so.